### ISSUE TITLE
No filename in error message

### ISSUE THREAD

**Issue by deakjahn** [localizely/flutter-intl-intellij#85]
During build I get the folllowing error message:

```
ERROR: Failed to generate localization files.
FormatException: Unexpected character (at line 5, character 1)
}
^

pub finished with exit code 2
```

It would be helpful to know which file the error is in. :-)

**Comment by lzoran** [comment]
Hi @deakjahn,

Thanks for reporting this!

According to the error message, it looks like the problem is in one of the arb files.

I agree that it would be useful to have such info in case of failure, especially if we need to check a lot of arb files.

**Comment by deakjahn** [comment]
Yes, certainly, it was relatively easy to find (especially that I knew what I had edited earlier), still, a message without important information is a nuisance.
